packages/dataviews/src/components/dataviews-view-config/index.tsx

```diff
@@ -549,34 +549,32 @@ function _DataViewsViewConfig( {
 	setDensity: React.Dispatch< React.SetStateAction< number > >;
 	defaultLayouts?: SupportedLayouts;
 } ) {
-	const [ isShowingViewPopover, setIsShowingViewPopover ] =
-		useState< boolean >( false );
-
 	return (
 		<>
 			<ViewTypeMenu defaultLayouts={ defaultLayouts } />
-			<div>
-				<Button
-					size="compact"
-					icon={ cog }
-					label={ _x( 'View options', 'View is used as a noun' ) }
-					onClick={ () => setIsShowingViewPopover( true ) }
-				/>
-				{ isShowingViewPopover && (
-					<Popover
-						placement="bottom-end"
-						onClose={ () => {
-							setIsShowingViewPopover( false );
-						} }
-						focusOnMount
-					>
-						<DataviewsViewConfigContent
-							density={ density }
-							setDensity={ setDensity }
+			<Dropdown
+				popoverProps={ { placement: 'bottom-end', offset: 9 } }
```

Comment:
This could be extracted as a constant object, to avoid passing a new instance every re-render

Reply:
Also done in https://github.com/WordPress/gutenberg/pull/65373